cdap-ui/app/features/overview/templates/data-apps.html

```diff
@@ -4,7 +4,8 @@
       <h3 class="text-uppercase"> Data </h3>
       <div class="row">
         <div class="col-xs-12 col-sm-5">
-          <p> Most recent </p>
+          <p ng-if="streams.concat(datasets).length < 5"> Showing {{ streams.concat(datasets).length }} out of {{ streams.concat(datasets).length }}</p>
```

Comment:
we're doing streams.concat(datasets) 5 times in these 2 lines.
Does it make sense to do the concatenation once in the controller and use that object in the 5 places here?